rank/scripts/updateIndex.ts

```diff
@@ -84,6 +84,26 @@ async function go() {
     error(`Couldn't reopen ${indexName} with error:`)
     console.info(openIndedxRes.error)
   }
+
+  const updateByQuery = await prompts({
+    type: 'confirm',
+    name: 'value',
+    message: 'Do you want to run the updateByQuery task now?',
+    initial: true,
+  }).then(({ value }) => value)
+
+  if (updateByQuery) {
+    const { body: updateByQueryRes } = await rankClient.updateByQuery({
+      wait_for_completion: false,
+      index: indexName,
+      conflicts: 'proceed',
+    })
+
+    const taskID = updateByQueryRes.task
+    success(`Update started successfully with task ID ${taskID}`)
+    info('You can monitor the task by running:')
+    code('yarn checkTask')
```

Comment:
Not for this patch perhaps, but this seems to lend itself to passing a task ID here, e.g.

```
yarn checkTask 289c9e59-318b-42cc-9fd4-763fb7e506a8
```

so you can inspect your task specifically, rather than finding it in the list of all tasks in the cluster.

Reply:
I think you're right here too - the automatic task discovery thing is nice, but users should really be able to specify the task they're checking. Maybe the best-of-both-worlds is to allow a task ID to be specified, but if it's not, give the user a choice from all available tasks.

I'll tidy this up in a separate PR.